catroidTest/src/org/catrobat/catroid/uitest/content/brick/VariableBricksTest.java

```diff
@@ -94,6 +97,40 @@ public void testVariableBricks() {
 				dataContainer.getUserVariable("p2", sprite).getValue());
 	}
 
+	public void testCreatingNewVariableCancelling() {
+		Spinner setVariableSpinner = solo.getCurrentViews(Spinner.class).get(0);
+		Spinner changeVariableSpinner = solo.getCurrentViews(Spinner.class).get(1);
+		UserVariable setValueVar = (UserVariable) setVariableSpinner.getSelectedItem();
+		UserVariable changeVariableVar = (UserVariable) changeVariableSpinner.getSelectedItem();
+		String spinnerText1 = setValueVar.getName();
+		String spinnerText2 = changeVariableVar.getName();
+
+		View view1 = solo.getView(Spinner.class, 0);
+		solo.clickOnView(view1);
+		solo.scrollToTop();
+		solo.clickOnView(solo.getView(TextView.class, 0));
+		solo.waitForText(solo.getString(R.string.cancel_button));
+		solo.clickOnButton(solo.getString(R.string.cancel_button));
+		solo.waitForText(spinnerText1);
+
+		View view2 = solo.getView(Spinner.class, 1);
+		solo.clickOnView(view2);
+		solo.scrollToTop();
+		solo.clickOnView(solo.getView(TextView.class, 0));
+		solo.waitForText(solo.getString(R.string.cancel_button));
+		solo.clickOnButton(solo.getString(R.string.cancel_button));
+		solo.waitForText(spinnerText2);
+
+		setValueVar = (UserVariable) setVariableSpinner.getSelectedItem();
+		changeVariableVar = (UserVariable) changeVariableSpinner.getSelectedItem();
+		String newSpinnerText1 = setValueVar.getName();
+		String newSpinnerText2 = changeVariableVar.getName();
+
+		assertTrue("Spinner has wrong value", spinnerText1.equals(newSpinnerText1));
+		assertTrue("Spinner has wrong value", spinnerText2.equals(newSpinnerText2));
+	}
+
+
```

Comment:
Since there is just one thing, the AS studio formatter and checkstyle nag about:
Remove this unnecessary empty line.